entity/page.php

```diff
@@ -482,6 +484,48 @@ public function set_template($template)
 	}
 
 	/**
+	* Get page icon font name
```

Comment:
That would be a job for a separate PR, as it would need to be done extension-wide.